### ISSUE TITLE
Deployment pod affinity not implemented

### ISSUE THREAD

**Issue by synhershko** [sl1pm4t/terraform-provider-kubernetes#44]
The following seems not to be supported for the deployment resource:

```hcl
        affinity {
          pod_anti_affinity {
            preferred_during_scheduling_ignored_during_execution {
              weight = 100
              pod_affinity_term {
                label_selector {
                  match_expressions {
                    key = "app"
                    operator = "In"
                    values = ["test"]
                  }
                }
              }
            }
          }
        }
```

This is the error we get:

```
Error: kubernetes_deployment.test: spec.0.template.0.spec.0: invalid or unknown key: affinity
```

Docs: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity

**Comment by kfirfer** [comment]
+1

**Comment by lomkju** [comment]
Waiting for this to get approved. Thanks @anuraaga

**Comment by sl1pm4t** [comment]
Fixed by #68
